src/pages/ShikshalokamVoiceChat/dynamic-voice-chat.js

```
@@ -0,0 +1,2757 @@
+              <MdAccountCircle />
+            </div>
+          )}
+          {userType === "bot" ? botNameToDisplay : name}
+        </div>
+        {!!message && !!recording && (
+          <div className={` ${userType === "bot" ? "div53" : "div54"} div50`}>
+            <WaveSurferPlayer url={recording?.result} {...default_wave_surfer_config} />
+          </div>
+        )}
+        <div className={` ${userType === "bot" ? "div53" : "div54"} div52 custom-voice-chat-chats`} id={chatId}>
+          <ReactMarkdown children={sanitizedContent} remarkPlugins={[remarkGfm]} rehypePlugins={[rehypeRaw]} className="prose max-w-none" />
```

Biome (2.3.14)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)